packages/framework/src/Component/Router/LocalizedRouterFactory.php

```diff
@@ -72,4 +131,13 @@ protected function getLocaleRouterResourceByLocale(string $locale): string
     {
         return str_replace('*', $locale, $this->localeRoutersResourcesFilepathMask);
     }
+
+    /**
+     * @param string $locale
+     * @return string
+     */
+    protected function getRoutingCacheDir(string $locale): string
+    {
+        return $this->cacheDir . '/routing/localized/' . $locale;
```

Comment:
We usually put paths in the specialized file (paths.yaml), or in services file directly. 
Having the directory configuration here is confusing as cacheDir is passed via constructor, but it's not the final cacheDir for routing.

Reply:
moved to `paths.yaml`